code/components/devtools-five/src/ArchetypeDebug.cpp

```diff
@@ -0,0 +1,292 @@
+#include <StdInc.h>
+#ifdef GTA_FIVE
```

Comment:
I wonder why you decided to guard this code? This is a `-five` component :P

Reply:
Removed 😅